mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1558,7 +1576,7 @@ def _log_eval_table(self):
                 data = self.dataset.features_data.assign(
                     **{
                         self.dataset.targets_name or "target": self.y,
-                        self.dataset.predictions_name or "outputs": self.y_pred,
+                        self.dataset.predictions_name or self.predictions or "outputs": self.y_pred,
```

Comment:
When will the `or "outputs"` be used?

Reply:
When its a single output from `model.predict` we don't know the column_name to use. Its the default. 

Example:

def predict(inputs):
    return [a, b]
    
    
There is no column_name here so we default it to `outputs`